pkg/plugins/layout.go

```diff
@@ -584,12 +584,24 @@ func (gd GdiskCall) GetLastSector(printOut string) (uint, error) {
 
 // Parses the output of a GdiskCall.Print call
 func (gd GdiskCall) GetSectorSize(printOut string) (uint, error) {
+
+	// Matching: "Logical sector size: 512 bytes"
 	re := regexp.MustCompile("sector size: (\\d+)")
 	match := re.FindStringSubmatch(printOut)
 	if match != nil {
 		size, err := strconv.ParseUint(match[1], 10, 0)
 		return uint(size), err
 	}
+
+	// Matching: "Sector size (logical/physical): 512/512 bytes"
+	// TODO: Why are there 2 different possible outputs from `sgdisk -p` ?
```

Comment:
Maybe that's related to the change in `1.0.2`:

```
On Linux (with 2.6.32 and later kernels), gdisk and sgdisk can now report both logical and physical sector sizes. This mainly affects the p/-p/--print display, but if you try to set an alignment value that's not a multiple of the ratio of the physical to logical sector size, the program will issue a warning. It also now uses the larger of the specified alignment value or this ratio when checking for improper alignment in the v/-v/--verify operation.
```

From here: https://www.rodsbooks.com/gdisk/revisions.html

On my VM I see:

```
localhost:~ # sgdisk --version
GPT fdisk (sgdisk) version 1.0.8
```

So I must have the newer sgdisk with the updated output.